<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalcExamlpe</class>
 <widget class="QMainWindow" name="CalcExamlpe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalcExamlpe</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(53, 106, 159);
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="3">
     <widget class="QPushButton" name="operation_div">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 229, 229);
border-color: rgb(0, 0, 0);
border-width: 2px;

border-style: inset;</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="operation_sqr">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 229, 229);
border-color: rgb(0, 0, 0);
border-width: 2px;

border-style: inset;</string>
      </property>
      <property name="text">
       <string>sqr</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="operation_minus">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 229, 229);
border-color: rgb(0, 0, 0);
border-width: 2px;

border-style: inset;</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="knopka_6">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 229, 229);
border-color: rgb(0, 0, 0);
border-width: 2px;

border-style: inset;</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="knopka_clear">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 229, 229);
border-color: rgb(0, 0, 0);
border-width: 2px;

border-style: inset;</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="knopka_4">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 229, 229);
border-color: rgb(0, 0, 0);
border-width: 2px;

border-style: inset;</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="knopka_3">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 229, 229);
border-color: rgb(0, 0, 0);
border-width: 2px;

border-style: inset;</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="knopka_1">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 229, 229);
border-color: rgb(0, 0, 0);
border-width: 2px;

border-style: inset;</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="b_plusOrMinus">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 229, 229);
border-color: rgb(0, 0, 0);
border-width: 2px;

border-style: inset;</string>
      </property>
      <property name="text">
       <string>+/-</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="operation_equals">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 229, 229);
border-color: rgb(0, 0, 0);
border-width: 2px;

border-style: inset;</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="operation_multi">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 229, 229);
border-color: rgb(0, 0, 0);
border-width: 2px;

border-style: inset;</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QLabel" name="result">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border-width: 3px;
border-style: inset;</string>
      </property>
      <property name="text">
       <string>Result: </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="knopka_0">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 229, 229);
border-color: rgb(0, 0, 0);
border-width: 2px;

border-style: inset;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="knopka_7">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 229, 229);
border-color: rgb(0, 0, 0);
border-width: 2px;

border-style: inset;</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="knopka_5">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 229, 229);
border-color: rgb(0, 0, 0);
border-width: 2px;

border-style: inset;</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QLabel" name="tablo">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border-width: 3px;
border-style: inset;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="knopka_8">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 229, 229);
border-color: rgb(0, 0, 0);
border-width: 2px;

border-style: inset;</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="operation_plus">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 229, 229);
border-color: rgb(0, 0, 0);
border-width: 2px;

border-style: inset;</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="operation_sqrt">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 229, 229);
border-color: rgb(0, 0, 0);
border-width: 2px;

border-style: inset;</string>
      </property>
      <property name="text">
       <string>sqrt</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="knopka_2">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 229, 229);
border-color: rgb(0, 0, 0);
border-width: 2px;

border-style: inset;</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="knopka_9">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 229, 229);
border-color: rgb(0, 0, 0);
border-width: 2px;

border-style: inset;</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="knopka_dot">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 229, 229);
border-color: rgb(0, 0, 0);
border-width: 2px;

border-style: inset;</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
